apps/desktop/electron-builder-ms.config.js

```
@@ -1,64 +1,24 @@
         'to': 'bin/bridge',
       },
     ],
     'extraFiles': [...DLLs],
-    'icon': 'build/static/images/icons/512x512.png',
+    'icon': 'app/build/static/images/icons/512x512.png',
     'artifactName': 'OneKey-Wallet-${version}-win-store-${arch}.${ext}',
     'verifyUpdateCodeSignature': false,
     'target': [{ target: 'nsis', arch: ['x64', 'arm64'] }],
-  },
-  'afterSign': 'scripts/notarize.js',
+  }
```

ESLint

[error] Insert `,`

(prettier/prettier)